src/templates/snek/gql/tasks/auth.ts

```diff
@@ -0,0 +1,142 @@
+//#region > Imports
+//> Parent Task
+// Contains the SNEK parent task
+import SnekTasks from "./index";
+//> Interfaces
+// Contains the user interface for authentication
+import { User } from "../../../../session";
+//> Types
+// Contains the type declaration for apollo results
+import { ApolloResult } from "./index";
+//#endregion
+
+//#region > Interfaces
+/**
+ * @interface AuthData defines the structure of the authentication result data
+ */
+interface AuthData {
+  auth: {
+    token: string;
+    refreshToken: string;
+    user: {
+      username: string;
+    };
+  };
+}
+
+/**
+ * @interface RefreshData defines the structure of the refresh result data
+ */
+interface RefreshData {
+  refresh: {
+    payload: string;
+    token: string;
+    refreshToken: string;
+  };
+}
+
+/**
+ * @interface RevokeData defines the structure of the revoke result data
+ */
+interface RevokeData {
+  revoke: {
+    revoked: string;
+  };
+}
+//#endregion
+
+//#region > Classes
+/** @class A set of session aware tasks */
+class SnekGqlAuthTasks {
+  /**
+   * @constructor
+   * @author Nico Schett <contact@schett.net>
+   * @param {string} session A session for the tasks
+   */
```

Comment:
Should have some sort of comment.

Initializes session aware SnekGqlAuth Task.

Reply:
Yeah, but not your suggestion cause this includes too less/useless information.
We have to discuss about a new TONIC rule in snek-at/tonic/issues/3